packages/tre/src/runtime/config/workerd.capnp.d.ts

```diff
@@ -12,6 +12,7 @@ export declare class Config extends __S {
     };
     static _Services: capnp.ListCtor<Service>;
     static _Sockets: capnp.ListCtor<Socket>;
+    static _Extensions: capnp.ListCtor<Extension>;
```

Comment:
I kinda feel like this file shouldn't be committed. Could it be generated as part of the `dev`/`build` process so that it's always in sync with `workerd.capnp`?

Reply:
Fair... We should investigate if there's a better way to do this in the near future